content/200-orm/050-overview/500-databases/600-mongodb.mdx

```diff
@@ -33,11 +33,11 @@ Some aspects of using Prisma ORM with MongoDB are the same as when using Prisma
 - connect to your database, using the [`mongodb` database connector](/orm/overview/databases)
 - use [Introspection](/orm/prisma-schema/introspection) for existing projects if you already have a MongoDB database
 - use [`db push`](/orm/prisma-migrate/workflows/prototyping-your-schema) to push changes in your schema to the database
-- use [Prisma Client](/orm/prisma-client) in your application to query your database in a type safe way based on your Prisma Schema
+- use [Prisma Client](/orm/prisma-client) in your application to query your database in a type safe way based on your Prisma schema
```

Comment:
i am pretty sure when referencing the schema in this way the S should be capitalized.  see https://www.notion.so/prismaio/Product-Naming-Guidelines-452abcc393734dd69a28fa8617e40ee6?pvs=4#e6fc78b306094c20903d66d48a4ada5c

Reply:
Will have to revert this change in multiple places

Reply:
@janpio could you confirm here?

Reply:
Linked document reflects current state I am aware of.